Boost Ctrl/Boost Ctrl/Controller/Matches/MatchesContentViewController.swift

```diff
@@ -188,6 +217,7 @@ class MatchesContentViewController: UIViewController {
 				} else {	// SAM RLCS is 5
 					matchesArrayRLCS[3].append(match)
 				}
+                matchesArray = matchesArrayRLCS
```

Comment:
This doesn't quite work right. Try going to the RLRS tab and doing a pull to refresh. The RLRS tab is then populated with RLRS data. If we add a similar line to `loadRLRSData()`, and refresh the RLCS tab, RLRS data is shown.